### DISCUSSION TITLE
Is it possible to remove the TOC in the blog?

### DISCUSSION THREAD

**Discussion by hhimanshu** [facebook/docusaurus#5305] [Q&A] [answered]
<img width="1416" alt="Screen Shot 2021-08-05 at 12 46 36 PM" src="https://user-images.githubusercontent.com/6589036/128411893-c0a6f417-1da2-4282-9788-162cb957467d.png">

To remove the sidebar we can say `blogSidebarCount: 0` and it goes away, but what about if I want to remove the TOC on the right hand side? Is there a setting that I can use to override? I searched [documentation](https://docusaurus.io/docs/markdown-features/inline-toc) and did not find anything relevant there. Please let me know.

**Comment by Siemienik** [comment, marked as answer]
The easiest way is to hide it by css.
Second, you turn it off for specific pages by using header option: `hide_table_of_contents: true`

**Reply by amelo1990** [reply]
hide_table_of_contents: true

**Reply by hhimanshu** [reply]
Thank you. This is great help. I can at least create a template for all my blog with `hide_table_of_contents: true` without fiddling with CSS.

**Reply by slorber** [reply]
another option is to swizzle `@theme/BlogLayout` and avoid rendering the TOC.

**Reply by slorber** [reply]
also related https://github.com/facebook/docusaurus/discussions/5220

**Reply by hhimanshu** [reply]
Thanks @slorber , did #5220 ever got implemented? I am interested in using a setting than to `swizzle`. I tried something yesterday, but I failed at that experiment :(

**Reply by amelo1990** [reply]
> Am einfachsten ist es, es per CSS auszublenden.
> Zweitens schalten Sie es für bestimmte Seiten aus, indem Sie die Kopfzeilenoption verwenden:`hide_table_of_contents: true`

**Reply by Siemienik** [reply]
I rather attempt to do not to use swizzle as long as possible for the reason of losing the possibility to upgrade in the future
